### DISCUSSION TITLE
How to change base URL "/admin" to "/" ?

### DISCUSSION THREAD

**Discussion by tk-subgenius** [filamentphp/filament#6109] [Help] [answered]
Hello,
  Please I'd appreciate if someone can give suggestions here..

  I'd like to setup the filament admin app on it's own site, so there would be:
 
  admin.<sitename> (Filament Admin)
  www.<siteName> (frontend site)

Is it possible to map "/admin" to "/" for 'admin.<sitename>'?  I could use URL rewriting with the web server, but looking for something built-in here.. 

Thank you!
Tom

**Comment by danharrin** [comment, marked as answer]
It's in the config file, `path` and `domain`

**Comment by tk-subgenius** [comment]
Thank you @danharrin !  I forgot to generate a filament config file, so didn't see that. 
FYI to other newbies, you need to remove/comment out the default '/' Route generated for your laravel project in routes/web.php:

```
// Route::get('/', function () {
//     return view('welcome');
// });
```

**Comment by ouziine** [comment]
How about the filament Version 3 
i want to change the base url i want /app instead of /admin

**Reply by andrewkhartley** [reply]
Also looking for this: I tried the following, but I may not have the syntax correct:

`'path' => 'bridge',`

**Reply by danharrin** [reply]
You do it in the panel provider file. This is the exact example in the docs.

**Reply by andrewkhartley** [reply]
I just did a fresh instance of Version ^3.0-stable. I ran this command:

`php artisan vendor:publish --tag=filament-config`

It produced this:

`return [

    /*
    |--------------------------------------------------------------------------
    | Broadcasting
    |--------------------------------------------------------------------------
    |
    | By uncommenting the Laravel Echo configuration, you may connect Filament
    | to any Pusher-compatible websockets server.
    |
    | This will allow your users to receive real-time notifications.
    |
    */

    'broadcasting' => [

        // 'echo' => [
        //     'broadcaster' => 'pusher',
        //     'key' => env('VITE_PUSHER_APP_KEY'),
        //     'cluster' => env('VITE_PUSHER_APP_CLUSTER'),
        //     'forceTLS' => true,
        // ],

    ],

    /*
    |--------------------------------------------------------------------------
    | Default Filesystem Disk
    |--------------------------------------------------------------------------
    |
    | This is the storage disk Filament will use to put media. You may use any
    | of the disks defined in the `config/filesystems.php`.
    |
    */

    'default_filesystem_disk' => env('FILAMENT_FILESYSTEM_DISK', 'public'),

];`

I'm not sure if I missed a step.

**Reply by pboivin** [reply]
https://filamentphp.com/docs/3.x/panels/configuration#changing-the-path

**Reply by andrewkhartley** [reply]
I see it now! Thank you @danharrin and @pboivin!

**Comment by skiarsi** [comment]
go to your application/app/Providers/Filament/AdminPanelProvider.php

in the panel function change ->path('panel') to ->path('app')

**Comment by david-huanca** [comment]
routes/web.php
```
// Route::view('/', 'welcome');
Route::redirect('/', '/admin');

```
